frontend/packages/console-app/src/components/console-operator/ConsolePluginDescriptionDetail.tsx

```diff
@@ -8,10 +8,17 @@ const ConsolePluginDescriptionDetail: React.FC<DetailsItemComponentProps> = ({ o
   const pluginStore = usePluginStore();
   const pluginName = React.useMemo(() => obj?.metadata?.name, [obj?.metadata?.name]);
 
-  const pluginInfo = React.useMemo(() => pluginStore.findDynamicPluginInfo(pluginName), [
-    pluginStore,
-    pluginName,
-  ]);
+  const pluginInfo = React.useMemo(
```

Comment:
The proper way to consume dynamic plugin runtime info is using `useDynamicPluginInfo` React hook which follows the reactive approach of re-render upon data change.

```ts
const [pluginInfoEntries] = useDynamicPluginInfo();
const pluginInfoEntry = React.useMemo(() => pluginInfoEntries.find((entry) => /* .. */), [pluginInfoEntries]);
```

Since we're renaming `PluginStore` function `getDynamicPluginInfo` to `getPluginInfo` we should also rename the hook `useDynamicPluginInfo` to `usePluginInfo` for better consistency.